thetis/coordsys.py

```diff
@@ -13,6 +13,7 @@
     datum='WGS84',
     units='m',
     errcheck=True)
+COORDSYS = UTM_ZONE10
```

Comment:
Good points. I made a start on changing to providing the coordinate system, rather than a `to_latlon` function. However, some of the forcing/interpolation functions now end up getting passed this twice, so they will need redesigning a bit.